appveyor.yml

```diff
@@ -0,0 +1,23 @@
+version: 2.0.0-beta04-{build}
+before_build:
+  - ps: Invoke-WebRequest "https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1" -OutFile ".\dotnet-install.ps1"
+  - ps: .\dotnet-install.ps1 -Version 1.0.0-preview2-003121
+  - ps: Remove-Item .\dotnet-install.ps1
+build_script:
+  - ps: dotnet --info
+  - ps: dotnet restore
+  - ps: dotnet build -c Debug .\*\project.json
+  - ps: dotnet build -c Release .\*\project.json
+before_test:
+  - ps: New-Item -ItemType Directory -Path tools -Force
+  - ps: Invoke-WebRequest "https://dist.nuget.org/win-x86-commandline/latest/nuget.exe" -OutFile ".\tools\nuget.exe"
```

Comment:
It appears nuget is already available in appveyor, so no need to install? https://www.appveyor.com/docs/nuget/#restoring-nuget-packages-before-build